vault/testing.go

```diff
@@ -2078,7 +2078,6 @@ func (m *mockBuiltinRegistry) Keys(pluginType consts.PluginType) []string {
 		"mysql-aurora-database-plugin",
 		"mysql-rds-database-plugin",
 		"mysql-legacy-database-plugin",
-
```

Comment:
This newline might have been intentional.

Reply:
Re-added